.github/CODEOWNERS

```diff
@@ -29,7 +29,21 @@
 # directory at the root of the repository and any of its
 # subdirectories.
 #/build/logs/ @doctocat
-/tests/manage/mcg/ @red-hat-storage/mcg-reviewers
+/tests/manage/z_cluster/ @red-hat-storage/brown-squad
+/tests/manage/pv_services/ @red-hat-storage/green-squad
+/tests/manage/storageclass/ @red-hat-storage/green-squad
+/tests/manage/monitoring/ @red-hat-storage/blue-squad
+/tests/manage/mcg/ @red-hat-storage/red-squad
+/tests/manage/rgw/ @red-hat-storage/red-squad
+/tests/manage/z_cluster/test_must_gather.py @red-hat-storage/purple-squad
```

Comment:
please delete this as the `test_must_gather.py` is already covered as part of brown squad in https://github.com/red-hat-storage/ocs-ci/pull/4265/files#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7R32

Reply:
Are we sure we wouldn't want the purple squad to be in charge of this particular file, and brown taking the rest?

Reply:
In `constants.py` I still see must gather assigned to Purple squad:

https://github.com/red-hat-storage/ocs-ci/blob/4fff1607a2124129777c0bc9b4bce1f4ac1f936e/ocs_ci/ocs/constants.py#L1260

Reply:
right..it's not intentional, we need to fix it there too